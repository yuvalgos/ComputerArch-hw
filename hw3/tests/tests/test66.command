./dflow_calc tests/test66.dat tests/test66.in d34 p107 p162 d95 d148 p138 d160 d178 p39 d58 p152 p136 d180 p35 d16 p8 d81 p189 d179 d59 p58 d176 d1 p20 p131 p112 d196 p169 d39 p78 d28 d59 d140 d31 p106 d93 p87 p31 p107 p37 d34 d10 p121 d8 d64 p170 p178 p194 d8 p56 d67 d164 d108 d94 d153 d140 p66 d135 d106 d146 p0 p67 d179 d194 d68 p103 p87 d70 p76 d112 d25 d127 d142 p2 p173 d102 d100 d149 p165 p20 d50 d196 d72 p39 d194 p161 p45 p134 d8