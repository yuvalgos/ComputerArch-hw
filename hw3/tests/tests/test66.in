7 0 21 7
8 12 31 19
6 30 20 12
8 1 2 23
6 6 27 15
7 21 22 27
2 24 15 31
1 11 7 19
1 3 16 22
0 28 16 30
2 9 25 27
6 10 20 13
8 17 5 27
0 11 20 30
6 29 0 19
7 21 16 6
1 25 5 13
8 4 21 18
5 2 9 30
2 30 27 4
4 7 16 24
0 8 16 9
4 3 14 13
1 25 29 14
0 21 15 27
0 19 14 14
6 15 8 2
4 31 11 11
7 1 18 21
5 11 29 29
5 0 7 22
8 26 7 26
1 15 12 5
2 7 25 18
0 8 18 2
0 31 0 26
5 24 25 31
5 10 31 26
3 11 8 23
8 19 8 18
5 10 26 1
0 4 27 30
0 23 28 10
0 18 19 3
2 18 11 29
2 3 17 5
4 10 20 21
4 15 24 1
8 17 29 20
0 19 21 3
3 17 21 14
6 26 6 29
0 31 8 30
7 24 8 21
6 30 16 1
4 6 22 4
0 3 19 20
1 29 22 15
7 29 12 7
4 6 18 14
0 10 10 9
2 27 0 0
0 12 6 14
6 20 5 8
3 30 30 25
4 6 21 6
8 23 21 16
8 12 8 31
0 8 13 24
5 4 10 25
4 29 13 8
3 20 23 0
4 13 29 2
8 21 31 9
5 0 8 16
5 19 22 3
5 22 7 1
0 30 13 4
8 17 20 31
0 28 4 16
8 25 20 23
6 0 19 14
3 20 1 13
0 9 25 17
7 21 28 15
5 9 12 11
1 30 19 14
3 31 5 30
7 11 9 16
6 0 4 3
2 21 15 15
3 30 11 3
8 29 18 15
8 24 22 30
1 13 27 16
0 21 22 19
4 29 12 16
3 23 19 15
1 27 7 5
0 29 31 28
6 24 18 18
7 28 6 26
6 26 16 1
2 21 28 14
4 0 1 29
4 18 14 19
2 28 2 10
0 16 24 1
0 24 7 12
6 31 2 6
0 0 7 25
0 20 7 16
0 25 20 17
0 28 30 21
0 0 5 24
4 26 28 29
2 7 9 27
0 10 21 2
7 28 18 4
2 6 20 26
7 4 22 20
0 14 16 0
8 8 18 28
3 3 28 12
7 9 18 6
4 28 16 8
8 1 23 29
3 7 31 11
1 23 27 25
0 30 14 19
6 19 30 17
0 15 29 17
0 21 27 18
2 1 9 21
3 28 18 28
7 19 3 5
0 30 10 22
4 8 3 17
8 21 10 21
8 5 29 11
0 21 2 19
8 21 17 24
4 24 4 16
0 17 4 16
4 17 23 15
8 1 23 28
5 25 17 25
8 30 5 20
4 21 7 6
4 3 2 28
7 4 30 11
0 27 11 3
2 25 19 16
6 4 2 2
2 13 18 8
5 9 11 14
4 8 18 15
3 1 10 1
6 5 13 31
2 21 26 11
8 0 12 3
4 16 27 0
1 0 23 17
4 12 28 4
5 4 18 14
6 19 16 8
1 21 13 14
5 10 24 4
2 4 14 12
0 25 14 24
5 8 31 0
8 3 6 16
6 26 13 5
5 20 11 2
5 6 22 14
7 19 3 26
8 2 8 30
6 17 25 30
6 28 12 4
4 28 31 6
2 23 12 8
3 20 5 15
6 7 0 17
5 14 12 4
1 8 18 1
2 25 11 23
1 9 6 15
1 24 18 0
1 24 21 22
0 26 9 9
6 13 9 14
4 29 31 6
4 29 28 25
0 22 21 29
4 31 15 12
5 27 26 8
1 14 4 1
2 10 7 15
6 24 5 21
6 4 13 5
