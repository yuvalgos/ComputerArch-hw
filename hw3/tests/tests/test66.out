Reading the opcodes latency info from tests/test66.dat ... Got latency for 9 opcodes
Reading the program file tests/test66.in ... Found 200 instructions
getProgDepth()==206
getInstDeps(34)=={-1,18}
getDepDepth(107)==84
getDepDepth(162)==105
getInstDeps(95)=={76,75}
getInstDeps(148)=={127,119}
getDepDepth(138)==115
getInstDeps(160)=={67,149}
getInstDeps(178)=={163,168}
getDepDepth(39)==55
getInstDeps(58)=={1,33}
getDepDepth(152)==115
getDepDepth(136)==115
getInstDeps(180)=={163,170}
getDepDepth(35)==39
getInstDeps(16)=={-1,-1}
getDepDepth(8)==0
getInstDeps(81)=={75,-1}
getDepDepth(189)==111
getInstDeps(179)=={109,174}
getInstDeps(59)=={44,-1}
getDepDepth(58)==44
getInstDeps(176)=={170,147}
getInstDeps(1)=={-1,-1}
getDepDepth(20)==5
getDepDepth(131)==89
getDepDepth(112)==89
getInstDeps(196)=={168,157}
getDepDepth(169)==118
getInstDeps(39)=={34,-1}
getDepDepth(78)==66
getInstDeps(28)=={-1,24}
getInstDeps(59)=={44,-1}
getInstDeps(140)=={18,135}
getInstDeps(31)=={20,-1}
getDepDepth(106)==53
getInstDeps(93)=={76,91}
getDepDepth(87)==48
getDepDepth(31)==16
getDepDepth(107)==84
getDepDepth(37)==41
getInstDeps(34)=={-1,18}
getInstDeps(10)=={-1,-1}
getDepDepth(121)==93
getInstDeps(8)=={-1,-1}
getInstDeps(64)=={54,23}
getDepDepth(170)==108
getDepDepth(178)==132
getDepDepth(194)==127
getInstDeps(8)=={-1,-1}
getDepDepth(56)==30
getInstDeps(67)=={34,52}
getInstDeps(164)=={105,121}
getInstDeps(108)=={33,67}
getInstDeps(94)=={61,-1}
getInstDeps(153)=={18,18}
getInstDeps(140)=={18,135}
getDepDepth(66)==2
getInstDeps(135)=={123,-1}
getInstDeps(106)=={18,60}
getInstDeps(146)=={144,112}
getDepDepth(0)==0
getDepDepth(67)==66
getInstDeps(179)=={109,174}
getInstDeps(194)=={131,163}
getInstDeps(68)=={-1,53}
getDepDepth(103)==108
getDepDepth(87)==48
getInstDeps(70)=={-1,68}
getDepDepth(76)==44
getInstDeps(112)=={111,78}
getInstDeps(25)=={-1,-1}
getInstDeps(127)=={109,88}
getInstDeps(142)=={120,107}
getDepDepth(2)==23
getDepDepth(173)==121
getInstDeps(102)=={-1,28}
getInstDeps(100)=={44,44}
getInstDeps(149)=={18,134}
getDepDepth(165)==161
getDepDepth(20)==5
getInstDeps(50)=={24,-1}
getInstDeps(196)=={168,157}
getInstDeps(72)=={70,18}
getDepDepth(39)==55
getInstDeps(194)=={131,163}
getDepDepth(161)==150
getDepDepth(45)==23
getDepDepth(134)==109
getInstDeps(8)=={-1,-1}
